www/src/content/blog/2025-04-07-agents-sdk-playground.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Introducing the Fiberplane Playground for Cloudflare Agents"
+description: "We're launching an Agents Playground for the Cloudflare Agents SDK"
+slug: cloudflare-agents-playground
+date: 2025-04-07
+author: Fiberplane team
+tags:
+  - cloudflare
+  - agents
+  - playground
+  - services architecture
+---
+
+import { Aside, LinkCard } from "@astrojs/starlight/components";
+import PackageManagers from "@/components/PackageManagers.astro";
+import { YouTube } from "astro-lazy-youtube-embed";
+
+Today we're releasing a new companion playground to help you build, test, and debug your agents on Cloudflare.
+
+The [Agents SDK](https://github.com/cloudflare/agents) on Cloudflare [launched](https://blog.cloudflare.com/build-ai-agents-on-cloudflare/) recently has made it straightforward to build, deploy, and scale AI agents directly at the edge using Workers and Durable Objects. The Fiberplane agents playground specifically addresses the challenges of debugging agents which can still be difficult:
+
+- they are inherently stateful and that state substantially influences LLM behavior;
+- they run on [Durable Objects](https://developers.cloudflare.com/durable-objects/), potentially spawning multiple instances.
+
+The Fiberplane team has had a lot of fun building agents, but we were lacking an overview of the agents themselves, and a way to debug them.
+
+## A bird's eye view of your agents
+
+The Fiberplane Agents [Playground](TODOLINKHERE) provides comprehensive visibility into all underlying state stored in each agent's Durable Object SQLite database, including state snapshots, messages, schedules, and tool calls with their full parameters and responses. This detailed introspection is captured separately for each instantiation of an agent, giving you complete clarity into individual agent behaviors.
```

Comment:
don't forget a link here

Reply:
whats the link inside the repo @flenter @laulauland